### DISCUSSION TITLE
NTP server problem

### DISCUSSION THREAD

**Discussion by markok12345** [tbnobody/OpenDTU#1782] [General]
My OpenDTU can't synchronize time from NTP server. It works before but I pushed accidentally manual sync push button and after that automatically synchronize has not worked after power off. I use original pool.ntp.org server. How I can fix this problem?

**Comment by markok12345** [comment]
Seems that I am only who have this problem. I did factory reset, but it doesn't help. What I can do, because this product is unusable with this problem? Can I use some cURL command what does time manual sync? This could be first help to this problem. Now OpenDTU working about one day and after this it lose connect to inverter although PV voltage is up

**Comment by markok12345** [comment]
Problem solved. If I turn DHCP off then NTP not work anymore. I switched DHCP back to on position and mada static IP address to my router. Now everything seems working well.

**Reply by ELeichstetten** [reply]
You are not alone. I have the same issue. 
Will try your solution.

**Reply by Blackout06** [reply]
Same here! NTP sync only works with DHCP ON

**Reply by tbnobody** [reply]
please provide a screenshot of you ntp config and network config.

**Reply by Blackout06** [reply]
<img width="660" alt="image" src="https://github.com/user-attachments/assets/7f23ecad-4f28-4519-bc09-720b913130cc" />
I have tried different server, including the default one, but synchronization only worked as soon as I static IP address.

**Reply by Blackout06** [reply]
<img width="521" alt="image" src="https://github.com/user-attachments/assets/cd8882cb-f90f-40b3-a8f6-fbed32026b97" />
Here is the network configuration.
Could it be linked to the DNS config ?

**Reply by tbnobody** [reply]
you try to reach the ntp server using a dns name but you dont specify a dns server. that cannot work. please specify a dns server or enter the IP of your ntp server.

**Reply by Blackout06** [reply]
Got it, Thanks a lot!!
I'm hopping that others had the same problem.
